utils/services/network/FirebaseApi.ts

```diff
@@ -325,6 +330,13 @@ export async function fbArticleAdd({
         // create new article with newly added thumbnail url
         const articleWithThumbnail = { ...article, thumbnail: thumbnailUrl };
 
+        // add some part of article to rtdb for searching purpose
+        try {
```

Comment:
can make a function out of this. Then call it

Reply:
Will do!